docs/UsingNavigators.md

```diff
@@ -78,7 +78,7 @@ render() {
     <Navigator
       initialRoute={{ title: 'My Initial Scene', index: 0 }}
       renderScene={(route, navigator) => {
-        <MyScene title={route.title} />
+        return <MyScene title={route.title} />
```

Comment:
Sure, that's a question of preference. I just tried to make as minimal changes as I could. Should I fix it up, @hramos ?